Documentation/supported-mod-list.md

```diff
@@ -429,286 +415,320 @@ Mods:
+125. [Island Bombing](https://twhl.info/competitions.php?results=13)
+126. [Ivy](https://twhl.info/vault.php?map=4869)
+127. [Jump Program](https://hl.loess.ru/?mod=575)
+128. [Killer on the Run](https://www.runthinkshootlive.com/posts/killer-on-the-run/)
+129. [Killing House for Half-Life](https://www.runthinkshootlive.com/posts/killing-house/)
+130. [Kosovo 2000](https://www.moddb.com/games/half-life/addons/kosovo-2000)
+131. [Kosovo II – The Second Day](htstp://www.moddb.com/games/half-life/addons/kosovo-2)
+132. [KotiteolliSuus](https://twhl.info/vault.php?map=533)
+133. [Kyo Half-Strike](https://csm.dev/threads/half-strike-rezultaty.36394/) (there are few error messages at start, which can be safely skipped; also the code in the beginning is 3141)
+134. [l33t Test 1: Conclusive Analysis](https://twhl.info/vault.php?map=3023)
+135. [Lab](https://twhl.info/vault.php?map=1797)
+136. [Lambda Station](https://www.runthinkshootlive.com/posts/lambda-station/)
+137. [LC](https://www.runthinkshootlive.com/posts/lc-by-sluxe/) (map for *Fast Level Design* competition by *Slux*)
+138. [LDSF( Laser Deployed Security Force )](https://twhl.info/competitions.php?results=4)
+139. [Locked Up](https://www.fileplanet.com/8842/0/fileinfo/Locked-Up)
+140. [Looping Stage](https://web.archive.org/web/20041104030218/http://cariad.co.za/twhl/mapvault_map.php?id=1250)
+141. [Lord's Lair](https://www.fileplanet.com/7471/0/fileinfo/Lords-Lair)
+142. [Losspower](https://twhl.info/vault.php?map=3919)
+143. [Lost-World](https://gamebanana.com/maps/156214)
+144. [Lounge](https://www.fileplanet.com/7461/0/fileinfo/The-Lounge)
+145. Matrixroom v0.7 (map by Joe Hunter)
+146. [MatrixTrainstation](https://twhl.info/vault.php?map=2531)
+147. [Maze01a](https://web.archive.org/web/20110512012317/http://www.richmans-maps.ch.vu/)
+148. [Meat, Blood, Gun](https://www.moddb.com/games/half-life/addons/meat-blood-gun) (there are 2 maps, but only first map is playable, in fact)
+149. [Merry Christmas to hlife_hotdog aka *Happy Holidays*](https://twhl.info/vault.php?map=5717) (to play the right map, enter map **happyholidays** into console, or edit *startmap* parameter in **liblist.gam** file inside mod's folder)
+150. Metro (map by unknown author)
+151. Monster Arena v1 & Monster Arena v2 (maps by unknown author)
+152. Monster Shoot (map by unknown author)
+153. [Museum Lockdown](https://twhl.info/vault.php?map=2211)
+154. [My Backyard](https://twhl.info/vault.php?map=1796)
+155. [My Black Mesa](https://www.runthinkshootlive.com/posts/my-black-mesa/) (map for *Single Mapping Competition* by *Zanzer*)
+156. [Nameless](https://www.runthinkshootlive.com/posts/nameless/) (map for *Single Mapping Competition* by *AGRESSOR*)
+157. [Need for Energy(SP-Energy)](https://scrama.3dn.ru/load/2-1-0-2)
+158. [Nightmare: A horror map](https://twhl.info/vault.php?map=2106)
+159. [Nightshift_FsC](https://twhl.info/vault.php?map=2561)
 150. [No Chance](https://gamebanana.com/maps/160536)
```

Comment:
It's 160.

Hey, this diff is quite difficult to read, maybe we can somehow get rid of numbered list or use some Markdown feature that created numbered list automatically?